homeassistant/components/deconz/cover.py

```diff
@@ -99,3 +110,26 @@ async def async_close_cover(self, **kwargs):
     async def async_stop_cover(self, **kwargs):
         """Stop cover."""
         await self._device.stop()
+
+    @property
+    def current_cover_tilt_position(self):
+        """Return the current tilt position of the cover."""
+        if self._device.tilt is not None:
+            return 100 - self._device.tilt
+
```

Comment:
We should explicitly `return None` in the other case.

Reply:
Ok! I'm working on next deCONZ PR so will add that

Reply:
Fixed in #43720